src/dal_select2/widgets.py

```diff
@@ -9,28 +9,55 @@
 
 from django import forms
 from django.utils import six
+from django.utils import translation
+from django.conf import settings
 
 
 class Select2WidgetMixin(object):
     """Mixin for Select2 widgets."""
 
-    class Media:
-        """Automatically include static files for the admin."""
+    def build_attrs(self, *args, **kwargs):
+        attrs = super(Select2WidgetMixin, self).build_attrs(*args, **kwargs)
+        lang_code = self._get_language_code()
+        if lang_code:
+            attrs.setdefault('data-autocomplete-light-language', lang_code)
+        return attrs
+
+    def _get_language_code(self):
+        # get_language can be return None
+        lang_code = translation.get_language()
+        if lang_code:
+            lang_code = translation.to_locale(lang_code).replace('_', '-')
+        return lang_code
 
-        css = {
-            'all': (
-                'autocomplete_light/vendor/select2/dist/css/select2.css',
-                'autocomplete_light/select2.css',
+    def _media(self):
+        """Automatically include static files for the admin."""
+        _min = '' if settings.DEBUG else 'min.'
+        i18n_file = ()
+        lang_code = self._get_language_code()
```

Comment:
For Russian language, `lang_code` here has value `'ru-RU'` and i18n file name is ru.js. So I had to replace this line with `lang_code = self._get_language_code().split('-',1)[0]` 
and then it worked for me. Otherwise, the file could not be loaded (error 404).